### DISCUSSION TITLE
zh/guide/install/desktop

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#174] [Announcements]
# zh/guide/install/desktop

import Desktop from 

https://alist.nn.ci/zh/guide/install/desktop.html

**Comment by 1121056478** [comment]
收费50，而且还不知道能不能用

**Reply by Mrered** [reply]
能用，但是有个小问题，macOS版运行的时候始终会在程序坞显示一个图标。

**Comment by yaodmin** [comment]
https://github.com/alist-org/alist/releases/tag/v3.13.2  到这去下

**Comment by A-wei-wei-del** [comment]
ubuntu 2204 缺少依赖libssl.so.1.1
